docs/intro-to-dune/index.md

```diff
@@ -343,12 +330,40 @@ That's because putting `reason-react-ppx` in the `preprocess/pps` field will
 transform function calls to `div` (which isn't defined anywhere) into calls to
 `React.createElement("div", ...)`[^2].
 
-<b>3.</b> The path of the `Hello.js` file generated from `foo/bar/Hello.re` is:
+:::
+
+<b>3.</b> Assume you have a directory `foo/bar` in the root of your project
+directory with these files in it:
+
+```
+foo/bar
+├─ Hello.re
+└─ dune
+```
+
+The contents of `foo/bar/dune` are:
+
+```dune
+(melange.emit
```

Comment:
The dune syntax in vscode-ocaml doesn't support melange.emit yet. It would be great to add the improvement and contribute it upstream.

Reply:
I added basic melange.emit support in https://github.com/melange-re/melange-for-react-devs/pull/29/commits/cdda37744774b417f83a02317965b159062219da. However, I didn't try to add any fields I haven't used yet, that's a rabbit hole I don't want to jump down at the moment.